rules/ios-prs.ts

```diff
@@ -0,0 +1,27 @@
+import { schedule, danger, fail, warn } from "danger"
+
+// The inspiration for this is https://github.com/artsy/artsy-danger/blob/f019ee1a3abffabad65014afabe07cb9a12274e7/org/all-prs.ts
+const isJest = typeof jest !== "undefined"
+
+// Stores the parameter in a closure that can be invoked in tests.
+const _test = (reason: string, closure: () => void | Promise<any>) =>
+  // We return a closure here so that the (promise is resolved|closure is invoked)
+  // during test time and not when we call wrap().
+  () => (closure instanceof Promise ? closure : Promise.resolve(closure()))
+
+// Either schedules the promise for execution via Danger, or invokes closure.
+const _run = (reason: string, closure: () => void | Promise<any>) =>
+  closure instanceof Promise ? schedule(closure) : closure()
+
+export const wrap: any = isJest ? _test : _run
+
+export const viewFilesWereChanged = wrap("View-ish files changes requires a screenshot ", () => {
+  const extensions = [".xib", ".storyboard", "View.swift", "Button.swift"]
```

Comment:
I think it's ok according to the way our project is today but it would be better to make it detect changes by the folder/package. So that if I add a custom "Label.swift" tomorrow, it will warn me as well.

Reply:
Yeah, I thought about that too. The downside of using packages is that we force every project to have the same folder structure, right? Not sure if that would be too much.

Another approach would be using a regex like `/\/views?\//i`, which would assume every view file belongs to a view folder. Not optimal yet but it does a better job.

What do you think?

Reply:
👍 to regex approach, then we can check both options :)

Reply:
It's fine for iOS projects I guess.